godtools/App/DependencyContainer/AppDataLayerDependencies.swift

```diff
@@ -33,7 +33,13 @@ class AppDataLayerDependencies {
     }
     
     // MARK: - Data Layer Classes
-            
+    
+    func getAccountInterfaceStringsRepositoryInterface() -> GetAccountInterfaceStringsRepositoryInterface {
```

Comment:
This is the best place here for this dependency.  However, I still need to spend some time on organizing the features.  There is an AccountCreation feature, but now I'm wondering if we should just have a general Account Feature for account related things.